examples/qualcomm/CMakeLists.txt

```diff
@@ -11,40 +11,88 @@ endif()
 cmake_minimum_required(VERSION 3.19)
 project(qualcomm_runner_example)
 
+if(NOT PYTHON_EXECUTABLE)
+  set(PYTHON_EXECUTABLE python3)
+endif()
 # Source root directory for executorch.
 if(NOT EXECUTORCH_ROOT)
   set(EXECUTORCH_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../..)
 endif()
+# Source root directory for pytorch.
+if(NOT TORCH_ROOT)
+  set(TORCH_ROOT ${EXECUTORCH_ROOT}/third-party/pytorch)
+endif()
 
-include(${EXECUTORCH_ROOT}/build/Utils.cmake)
+# Find prebuilt libraries. executorch package should contain
+# portable_ops_lib, etdump, bundled_program.
+find_package(executorch CONFIG REQUIRED)
+find_package(gflags REQUIRED)
+find_library(android_log log)
+set(qnn_lib_list
+    qnn_backend
```

Comment:
This is quite hard to maintain. Maybe you can build and install a shared library and depend on that?

Reply:
Thank you for the suggestion, changed. Also update example scripts to make them ship `lib_qnn_executorch_backend.so` as well.